File: CS/SpellCheckerCustomization/Models/MyHtmlEditorExtension.cs
using DevExpress.Web.Mvc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using DevExpress.Web.ASPxHtmlEditor.Internal;
using System.Web.UI;
using DevExpress.Web.ASPxSpellChecker.Forms;
using DevExpress.Web;
using DevExpress.Web.Mvc.UI;
using System.Web.Mvc;

namespace SpellCheckerCustomization.Models {
    public class MyHtmlEditorExtension : HtmlEditorExtension {
        public MyHtmlEditorExtension(HtmlEditorSettings settings, ViewContext ctx)
            : base(settings, ctx) {
        }
        protected override ASPxWebControl CreateControl() {
            return new MVCxHtmlEditor();
        }
    }
    public class MVCxHtmlEditor : DevExpress.Web.Mvc.MVCxHtmlEditor {
        protected override DevExpress.Web.ASPxHtmlEditor.Internal.HtmlEditorSpellChecker CreateSpellCheckerInstance() {
            MVCxHtmlEditorSpellChecker check = new MVCxHtmlEditorSpellChecker(this);
            return check;
        }
    }

    class MVCxHtmlEditorSpellChecker : DevExpress.Web.Mvc.MVCxHtmlEditorSpellChecker {
        public MVCxHtmlEditorSpellChecker(DevExpress.Web.Mvc.MVCxHtmlEditor htmlEditor) : base(htmlEditor) { }
        protected override Control CreateSpellCheckForm() {
            return new SpellCheckForm();
        }
    }

    class SpellCheckForm : DevExpress.Web.ASPxSpellChecker.Forms.SpellCheckForm {
        protected override void PrepareChildControls() {
            base.PrepareChildControls();
            // Customize controls here
            this.btnIgnore.ControlStyle.CssClass = "btn-custom";
            this.btnAddToDictionary.Text = "Add";
        }
    }
}

File: VB/SpellCheckerCustomization/Models/MyHtmlEditorExtension.vb
Imports DevExpress.Web.Mvc
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports DevExpress.Web.ASPxHtmlEditor.Internal
Imports System.Web.UI
Imports DevExpress.Web.ASPxSpellChecker.Forms
Imports DevExpress.Web
Imports DevExpress.Web.Mvc.UI
Imports System.Web.Mvc

Namespace SpellCheckerCustomization.Models
	Public Class MyHtmlEditorExtension
		Inherits HtmlEditorExtension

		Public Sub New(ByVal settings As HtmlEditorSettings, ByVal ctx As ViewContext)
			MyBase.New(settings, ctx)
		End Sub
		Protected Overrides Function CreateControl() As ASPxWebControl
			Return New MVCxHtmlEditor()
		End Function
	End Class
	Public Class MVCxHtmlEditor
		Inherits DevExpress.Web.Mvc.MVCxHtmlEditor

		Protected Overrides Function CreateSpellCheckerInstance() As DevExpress.Web.ASPxHtmlEditor.Internal.HtmlEditorSpellChecker
			Dim check As New MVCxHtmlEditorSpellChecker(Me)
			Return check
		End Function
	End Class

	Friend Class MVCxHtmlEditorSpellChecker
		Inherits DevExpress.Web.Mvc.MVCxHtmlEditorSpellChecker

		Public Sub New(ByVal htmlEditor As DevExpress.Web.Mvc.MVCxHtmlEditor)
			MyBase.New(htmlEditor)
		End Sub
		Protected Overrides Function CreateSpellCheckForm() As Control
			Return New SpellCheckForm()
		End Function
	End Class

	Friend Class SpellCheckForm
		Inherits DevExpress.Web.ASPxSpellChecker.Forms.SpellCheckForm

		Protected Overrides Sub PrepareChildControls()
			MyBase.PrepareChildControls()
			' Customize controls here
			Me.btnIgnore.ControlStyle.CssClass = "btn-custom"
			Me.btnAddToDictionary.Text = "Add"
		End Sub
	End Class
End Namespace